apps/pvx/restapi/custom/api.pm

```diff
@@ -166,6 +167,13 @@ sub query_range {
 
     my $result = $self->get_endpoint(url_path => '/query?expr=' . $uri->encode($query));
 
+    if (defined( $self->{option_results}->{default_value} ) && !exists( $result->{data} )) {
+        my $forced_hash;
+        $options{filter} =~ /([^\s\\]+) = \"([^\s\\]+)\"/;
+        push @{ $forced_hash->{key} }, { value => $2 };
+        push @{ $forced_hash->{values} }, { value => $self->{option_results}->{default_value} }; push @{$result->{data}}, $forced_hash;
```

Comment:
i don't understand your data structure with forced_hash. You put it directly in result->data

Reply:
Yes indeed